core/src/services/memcached/backend.rs

```diff
@@ -85,7 +85,7 @@ impl MemcachedBuilder {
     ///
     /// Will panic if `max_size` is 0.
     #[must_use]
-    pub fn connection_pool_max_size(mut self, max_size: u32) -> Self {
+    pub fn connection_pool_max_size(mut self, max_size: usize) -> Self {
```

Comment:
This is a breaking change but I think it's OK. Our next version would be thus 0.56.0.